packages/vscode-ext/package.json

```diff
@@ -93,7 +93,7 @@
     "dev": "NODE_ENV=development tsup --watch",
     "build": "NODE_ENV=production tsup",
     "lint": "eslint src --ext ts",
-    "pack": "vsce package",
+    "pack": "vsce package --no-dependencies",
```

Comment:
is this CLI option no harmful for the final production bundle?

Reply:
After my test, I found that this will not affect its function at all, and the size of the plug-in has also dropped from 5.41mb to 1.47mb.